mimeparse/__init__.py

```diff
@@ -4,14 +4,16 @@
 __license__ = 'MIT License'
 __credits__ = ''
 
+from typing import Dict, Generator, Iterable, Tuple, Union
```

Comment:
let's import also `from __future__ import annotations`

Reply:
what for? 🤔

Reply:
to enable postponed evaluation of the annotations. see https://peps.python.org/pep-0563/